Extensions/mirrorposts.js

```diff
@@ -0,0 +1,88 @@
+//* TITLE Mirror Button **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION	A button to easily backup posts to archive.is or archive.org **//
+//* DEVELOPER Legoben **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.mirrorposts = new Object({
+
+	running: false,
+
+	preferences: {
+		"archiveis": {
+			text: "Use archive.is instead of The WayBack Machine",
+			default: false,
+			value: false
+		},
+		"menu": {
+			text: "Archive posts through the share menu instead of having a button",
+			default: true,
+			value: false
```

Comment:
Seems a bit odd to have the default be true but the value be false in the code.  I have no idea if this will break anything or cause it to behave unexpectedly, just looks a bit odd.